ksqldb-functional-tests/src/test/resources/query-validation-tests/max-group-by.json

```diff
@@ -135,6 +135,108 @@
         {"topic": "S2", "key": 0,"value": {"VALUE": "20.99"}},
         {"topic": "S2", "key": 0,"value": {"VALUE": "20.99"}}
       ]
+    },
+    {
+      "name": "max date group by valid after 7.2.0",
```

Comment:
We probably should add QTTs for min as well.

Reply:
Yeah, pushing an addendum as we speak

Reply:
Awesome; great work to get the unit test and QTTs!